proxybroker/negotiators.py

```diff
@@ -61,6 +62,8 @@ async def negotiate(self, **kwargs):
         await self._proxy.send(struct.pack('3B', 5, 1, 0))
         resp = await self._proxy.recv(2)
 
+        if not isinstance(resp, (bytes, str)):
```

Comment:
Have you encountered any issues here?
Because the response comes from the python build-in `asyncio.StreamReader`, it should return the bytes object.